### DISCUSSION TITLE
change to ShouldSkipRecord in v28

### DISCUSSION THREAD

**Discussion by scouch1** [JoshClose/CsvHelper#2005] [General]
I have been using this config successfully until upgrade to v28.

var csvConfig = new CsvConfiguration(CultureInfo.InvariantCulture)
{
    ShouldSkipRecord = record => record.Record.All(field => String.IsNullOrWhiteSpace(field))
};

I found information about the change here: https://joshclose.github.io/CsvHelper/migration/v28/
which indicates string[] ShouldSkipRecordArgs.Record changed to IRecordRow ShouldSkipRecordArgs.Row

In my case, It appears to be IReaderRow not IRecordRow and even the sample provided (using length) does not work?

I'm obviously doing something wrong. Any suggestions would be appreciated.

**Comment by JoshClose** [comment]
Fixed. You might need to refresh the page for it to show.

**Reply by scouch1** [reply]
Awesome! Thanks for the quick update.
